docs/book/driver-deployment/prerequisites.md

```diff
@@ -0,0 +1,156 @@
+
+Roles needs to be assigned to the vSphere objects that participate in the Cloud Native Storage environment.
+
+To understand roles assigment to vSphere objects, consider we have following vSphere inventory.
+
+```bash
+sc2-rdops-vm06-dhcp-215-129.eng.vmware.com (vCenter Server)
+|
+|- datacenter (Data Center)
+    |
+    |-vSAN-cluster (cluster)
+      |
+      |-10.192.209.1 (ESXi Host)
+      | |
+      | |-k8s-master (node-vm)
+      |
+      |-10.192.211.250 (ESXi Host)
+      | |
+      | |-k8s-node1 (node-vm)
+      |
+      |-10.192.217.166 (ESXi Host)
+      | |
+      | |-k8s-node2 (node-vm)
+      | |
+      |-10.192.218.26 (ESXi Host)
+      | |
+      | |-k8s-node3 (node-vm)
+```
+
+Consider each host has following shared datastores along with some local VMFS datastores.
+
+- shared-vmfs
+- shared-nfs
+- vsanDatastore
+
+Considering above inventory, roles should be assigned as specified below.
+
+| Role  | Usage  |
+|---|---|
+| Read-Only | ![READ-ONLY-USAGE](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/READ-ONLY-USAGE.png)  |
+| CNS-HOST-CONFIG-STORAGE | ![HOST-CONFIG-STORAGE-USAGE](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/HOST-CONFIG-STORAGE-USAGE.png)   |
+| CNS-DATASTORE | ![CNS-DATASTORE-USAGE](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/CNS-DATASTORE-USAGE.png)  |
+| CNS-VM | ![CNS-VM-USAGE](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/CNS-VM-USAGE.png)  |
+| CNS-SEARCH-AND-SPBM | ![CNS-SEARCH-AND-SPBM-USAGE](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/CNS-SEARCH-AND-SPBM-USAGE.png)  |
+
+## Setting up the management network <a id="setup_management_network"></a>
+
+By default, CPI and CSI Pods are scheduled on k8s master nodes. In this case, for non-topology aware Kubernetes clusters, it is sufficient to provide the k8s master node(s) credentials to the vCenter that this cluster runs on.
+
+For topology aware clusters, every k8s node needs to discover its topology by communicating with the vCenter. This is needed to utilize the topology aware provisioning and late binding feature.
+
+Refer the CSI Deployment section to understand how to provide a kubernetes node with vCenter credentials.
```

Comment:
s/Refer the CSI Deployment/Refer to the CSI Deployment

Reply:
done.